src/setup/joystick.c

```diff
@@ -1133,6 +1139,34 @@ static void SwapLRSticks(TXT_UNCAST_ARG(widget), TXT_UNCAST_ARG(unused))
     }
 }
 
+static void AdjustAnalog(TXT_UNCAST_ARG(widget), TXT_UNCAST_ARG(unused))
+{
+    txt_window_t *window;
+
+    window = TXT_NewWindow("Analog Settings");
+    TXT_SetTableColumns(window, 2);
+    TXT_SetColumnWidths(window, 10, 6);
+    TXT_AddWidgets(window,
+        TXT_NewCheckBox("Use analog controls", &use_analog),
```

Comment:
How about turning it on by default?

Reply:
I was thinking the same thing. If we detect a gamepad, set `use_analog` to 1. However I was trying to stick to the Chocolate philosophy of defaulting to vanilla behavior.

Reply:
> If we detect a gamepad, set `use_analog` to 1.

I think it would fit Crispy Doom.

Reply:
Sure. Though, I think we should also go with reasonable defaults (at least for inputs) in Chocolate Doom.